### ISSUE TITLE
Incomplete / faulty TOR nodes list used

### ISSUE THREAD

**Issue by sebix** [MISP/MISP#4619]
Hello,

I have been made aware that MISP uses an in incomplete and wrong TOR (exit) node list.

At https://www.misp-project.org/feeds/#default-feeds-available-in-misp and in https://github.com/MISP/MISP/blob/2.4/app/files/feed-metadata/defaults.json#L189, https://github.com/MISP/MISP/blob/2.4/app/files/feed-metadata/defaults.json#L222 the list by **dan.me.uk** is used. This list is known to be wrong as it contains also non-exit nodes and does not have all the exit nodes. I can also confirm the latter.

There is an accurate list by the torproject itself: https://check.torproject.org/exit-addresses

Additionally there are lists and APIs which allow you to query _historic_ data: https://metrics.torproject.org/exonerator.html / https://metrics.torproject.org/rs.html

**Comment by adulau** [comment]
Indeed, good point. Thank you very much. There are some major changes ongoing in the feed parser to allow more flexibility. 

We will add https://check.torproject.org/exit-addresses very soon as the format is not correctly parsing in free-text.
